libs/tup-components/src/layout/Sidebar/Sidebar.module.css

```diff
@@ -1,5 +1,6 @@
 .root {
   overflow-y: auto;
+  height: 100%;
```

Comment:
**Note to Self:** @wesleyboar, this is only necessary because `<nav>` has an inner `<div>` (two actually), but removing that nesting requires excess trickery in at least two components (because PageLayout is an abstraction), so don't worry about it. One clear style is better than two or so React props that may only be used once.